repo.go

```diff
@@ -19,6 +19,9 @@ type Interface interface {
 	// Get returns the charm referenced by curl.
 	Get(curl *charm.URL) (charm.Charm, error)
 
+	// GetBundle returns the bundle referenced by curl.
+	GetBundle(curl *charm.URL) (charm.Bundle, error)
```

Comment:
Technically, we could have added GetBundle as an optional method
(accessible with a type assertion) but moving to v1 seems like
a good idea anyway and this is a good moment to expand the interface.